src/Test.FunctionalTests.BinSkim.Driver/CommandLineTests.cs

```diff
@@ -18,37 +18,26 @@ namespace Microsoft.CodeAnalysis.BinSkim.Driver
 {
     public class CommandLineTests
     {
-        private const string argsStringBase = @"analyze C:\Native_x64_VS2019_No_CastGuard.exe -o C:\result.sarif -f";
+        private const string argsStringBase = @"analyze C:\Native_x64_VS2019_No_CastGuard.exe -o C:\result.sarif --log ForceOverwrite";
 
         private class SarifVersionTestCase
         {
-            internal string InputBinSkimSarifVersion { get; }
             internal string ArgsString { get; }
-            internal Sarif.SarifVersion? ExpectedSarifVersion { get; }
             internal string ExpectedErrorParameter { get; }
 
-            internal SarifVersionTestCase(string inputBinSkimSarifVersion, Sarif.SarifVersion? expectedSarifVersion, string expectedErrorParameter)
+            internal SarifVersionTestCase(string expectedErrorParameter)
             {
-                this.InputBinSkimSarifVersion = inputBinSkimSarifVersion;
-                this.ArgsString = inputBinSkimSarifVersion == null ? argsStringBase : argsStringBase + inputBinSkimSarifVersion;
-                this.ExpectedSarifVersion = expectedSarifVersion;
+                this.ArgsString = argsStringBase;
                 this.ExpectedErrorParameter = expectedErrorParameter;
             }
         }
 
         [Fact]
-        public void CorrectlyParseGenerateJsonIntegerAs()
```

Comment:
>CorrectlyParseGenerateJsonIntegerAs [](http://example.com/codeflow?start=20&length=35)

Huh? This test didn't test json integer at all. We no longer permit SARIF v1 to be generated so nearly everything useful is gone from these tests. Leaving the code as a stub to build out in a more useful way later.